### PR TITLE
[MODORDSTOR-419] - Add user-tenants permission for kafka process

### PR DESCRIPTION
**Pull request by azizbekxm**
https://folio-org.atlassian.net/browse/MODORDSTOR-419
## Purpose:
- kafka consumer in mod-orders-storage fetch central tenant from user-tenants

### CODE REVIEW

**Review by BKadirkhodjaev — APPROVED** [review]

File: descriptors/ModuleDescriptor-template.json [hunk 1]
```diff
@@ -539,7 +542,8 @@
           "modulePermissions": [
             "inventory-storage.holdings.item.put",
             "inventory-storage.holdings.collection.get",
-            "inventory-storage.instances.item.get"
+            "inventory-storage.instances.item.get",
+            "user-tenants.collection.get"
```
Comment (SerhiiNosko) [inline descriptors/ModuleDescriptor-template.json:546]: this permission is used by mod-orders-storage Holding Create consumer in ECS mode:
https://github.com/folio-org/mod-orders-storage/blob/c0923e3a120d7108241e0ecc1f7a22cb7da246ae/src/main/java/org/folio/event/handler/HoldingCreateAsyncRecordHandler.java#L50

File: descriptors/ModuleDescriptor-template.json [hunk 2]
```diff
@@ -328,7 +328,9 @@
           "permissionsRequired": ["inventory.holdings.item.put"],
           "modulePermissions": [
             "inventory-storage.holdings.item.get",
-            "inventory-storage.holdings.item.put"
+            "inventory-storage.holdings.item.put",
+            "inventory-storage.holdings.collection.get",
+            "inventory-storage.holdings.batch.post"
```
Comment (SerhiiNosko) [inline descriptors/ModuleDescriptor-template.json:333]: These 2 permissions are used by Holding Update Kafka consumer that needs to fetch other holdings that using in the same order and update them to use the same instanceId. In this case all holdings in order will have holdings connected to the same instance:
https://github.com/folio-org/mod-orders-storage/blob/c0923e3a120d7108241e0ecc1f7a22cb7da246ae/src/main/java/org/folio/services/inventory/InventoryUpdateService.java#L47

File: descriptors/ModuleDescriptor-template.json [hunk 3]
```diff
@@ -328,7 +328,10 @@
           "permissionsRequired": ["inventory.holdings.item.put"],
           "modulePermissions": [
             "inventory-storage.holdings.item.get",
-            "inventory-storage.holdings.item.put"
+            "inventory-storage.holdings.item.put",
+            "inventory-storage.holdings.collection.get",
+            "inventory-storage.holdings.batch.post",
+            "user-tenants.collection.get"
```
Comment (SerhiiNosko) [inline descriptors/ModuleDescriptor-template.json:334]: "user-tenants.collection.get" is used by mod-orders-storage Item Update Kafka consumer in ECS mode:
https://github.com/folio-org/mod-orders-storage/pull/442/files#diff-f34125a4cad28f81be682f398500f5bb7697e4763bb10edbe8bc3a6d83423ca5R41

**Review by SerhiiNosko — APPROVED** [review]

File: descriptors/ModuleDescriptor-template.json [hunk 4]
```diff
@@ -307,7 +307,8 @@
             "inventory-storage.items.item.get",
             "inventory-storage.items.collection.get",
             "inventory-storage.items.item.put",
-            "users.item.get"
+            "users.item.get",
+            "user-tenants.collection.get"
```
Comment (SerhiiNosko) [inline descriptors/ModuleDescriptor-template.json:311]: this is used by orders-storage Item Update Kafka consumer to update releted piece(that can be located in another tenant)

**Review by RuslanLavrov — APPROVED** [review]

**Pull request merged**